src/App.js

```diff
@@ -11,14 +11,16 @@ import HomeScreen from './components/HomeScreen/HomeScreen';
 import Call from './components/Call/Call';
 import Header from './components/Header/Header';
 import Tray from './components/Tray/Tray';
+import HairCheck from './components/HairCheck/HairCheck';
 
-/* We decide what UI to show to users based on the state of the app, which is dependent on the state of the call object: see line 137. */
+/* We decide what UI to show to users based on the state of the app, which is dependent on the state of the call object. */
```

Comment:
Another "not super related" comment so feel free to make in a separate PR or whatever (just noticed while clicking around):
This resize looks a little funky to me. Generally we want to treat the waiting for others card as if it were a video tile, so I'd expect them to resize the same way.
![CleanShot 2022-05-13 at 14 22 27](https://user-images.githubusercontent.com/18265845/168344496-cce0ff5a-6d00-4aed-a734-b623bca3f36b.gif)

Reply:
Should be fixed now!